docker-compose.dev.yml

```diff
@@ -77,6 +77,8 @@ services:
       - UKEF_TFM_API_REPORTS_KEY
       - TFM_URI=http://localhost:5003
       - AZURE_ACBS_FUNCTION_URL=http://host.docker.internal:7071
+      - GOV_NOTIFY_API_KEY
+      - GOV_NOTIFY_EMAIL_RECIPIENT
```

Comment:
Is this needed?

Reply:
it won't send the email without it right? I need to send the email eventually to test it

Reply:
are we not using reference data proxy to send the emails though?

Reply:
true, I will remove this in the PR to fix emails

Reply:
https://ukef-dtfs.atlassian.net/browse/DTFS2-5478

Reply:
I don't understand why we're adding this env variable if it's only going to get removed in the next PR/ticket